uaclient/cli/vulnerability/__init__.py

```diff
@@ -0,0 +1,21 @@
+from uaclient import messages
+from uaclient.cli.commands import ProCommand
+from uaclient.cli.vulnerability.list import list_subcommand
+from uaclient.cli.vulnerability.show import show_subcommand
+from uaclient.cli.vulnerability.update import update_subcommand
+
+
+def action_vulnerability(args, *, cfg, **kwargs):
+    # Avoiding a circular import
+    from uaclient.cli import get_parser
+
+    get_parser().print_help_for_command("vulnerability")
+
+
+vulnerability_command = ProCommand(
```

Comment:
When creating this command, it's important to assign it to a `help_category` so it appears in the initial `pro --help` content. So far, it is shown as an option in the autocomplete, but is not listed in the help.
(I think this is `HelpCategory.SECURITY`)

Reply:
(This implying changing the `help`integration test to include this option there)